astro.config.mjs

```diff
@@ -94,26 +94,21 @@ export default defineConfig({
         { label: 'Reference',
           // translations: { 'fr': 'Référence', },
           items: [
-            { label: 'Tidal functions',
-              autogenerate: { directory: 'reference/tidal' },
-              collapsed: true,
-              badge: 'WIP',
+            { label: 'Tidal',
+              items: [
+                {label: 'Configuration | The boot file', link: 'reference/tidal/configuration' },
```

Comment:
Can we keep it with a colon, like `Configuration: the boot file` or just `Configuration`?